server/repository/src/db_diesel/invoice_line_row.rs

```diff
@@ -161,10 +187,18 @@ impl<'a> InvoiceLineRowRepository<'a> {
         Ok(())
     }
 
-    pub fn delete(&self, invoice_line_id: &str) -> Result<(), RepositoryError> {
+    pub fn delete(&self, invoice_line_id: &str) -> Result<Option<i64>, RepositoryError> {
+        let old_row = self.find_one_by_id(invoice_line_id)?;
+        let change_log_id = match old_row {
+            Some(old_row) => self.insert_changelog(&old_row, RowActionType::Delete)?,
```

Comment:
Do you think this i really needed ? Why not just insert any way ?

Reply:
If we didn't look up the old record, we don't get the `store_id` and `name_link_id` assigned? Is that ok for a Delete? Maybe it is? I wasn't sure so went this way :)